vendor/fbtransform/transforms/xjs.js

```diff
@@ -11,6 +11,51 @@
 var Syntax = require('jstransform').Syntax;
 var utils = require('jstransform/src/utils');
 
+function commaAfterLastParen(value) {
+    var state = "normal";
+    var commaPos = 0;
+    for (var i=0;i<value.length;++i) {
+        if (state === "normal") {
```

Comment:
@hzoo Yeah, I did that, but it didn't flag any of the style issues zpao pointed out.